agent/xds/testdata/rbac/v2-ignore-empty-permissions.golden

```diff
@@ -0,0 +1,22 @@
+{
+  "filters": [
+    {
+      "name": "envoy.filters.network.rbac",
+      "typedConfig": {
+        "@type": "type.googleapis.com/envoy.extensions.filters.network.rbac.v3.RBAC",
+        "rules": {
+          "action": "DENY"
+        },
+        "statPrefix": "connect_authz"
+      }
+    },
+    {
+      "name": "envoy.filters.network.rbac",
+      "typedConfig": {
+        "@type": "type.googleapis.com/envoy.extensions.filters.network.rbac.v3.RBAC",
+        "rules": {},
```

Comment:
The test case looks like default allow mode, but with an empty permissions, did this get switched to default deny mode?

Reply:
Yeah, since there is one permission (even though it's invalid and ignored) default deny is turned on. I didn't want to spend much time thinking about this since this is defensive against something that's impossible and is consistent with the way traffic permissions are working in general.

Reply:
By consistent I mean, a TP exists therefore default deny is turned on.

Reply:
Ahh got it